docs/operation/account_administration.rst

```diff
@@ -0,0 +1,88 @@
+
+Account Administration
+======================
+
+This section highlights **MetalK8s Account Administration** which covers
+changing the default username and password for some MetalK8s services.
+
+
+Administering Grafana
+*********************
+
+A fresh install of MetalK8s has a Grafana service instance with default
+credentials: ``admin`` / ``admin``. For more information on how to access
+Grafana, please refer to :ref:`this procedure <quickstart-services-grafana>`
+
+Changing Grafana username and password
+--------------------------------------
+
+To change the default username and password for Grafana on a MetalK8s cluster,
+perform the following procedures:
+
+#. Create a file named ``patch-secret.yaml`` that has the following content:
+
+   .. code-block:: yaml
+
+      stringData:
+        admin-password: <password-in-clear>
+        admin-user: <username-in-clear>
+
+#. Apply the patch file by running:
+
+   .. code-block:: shell
+
+      $ kubectl --kubeconfig /etc/kubernetes/admin.conf patch secrets prometheus-operator-grafana --patch "$(cat patch-secret.yaml)" -n metalk8s-monitoring
+
+#. Now, roll out the new updates for Grafana.
+
+   .. code-block:: shell
+
+      $ kubectl --kubeconfig /etc/kubernetes/admin.conf rollout restart deploy prometheus-operator-grafana -n metalk8s-monitoring
+
+#. Access the Grafana instance and authenticate yourself using the new Account
+   credentials.
+
+
+Administering MetalK8s GUI, Kubernetes(K8S) and Salt API
+********************************************************
+
+A fresh install of MetalK8s has a GUI instance with default credentials:
+``admin`` / ``admin``. For more information on how to access the GUI, please
+refer to :ref:`this procedure <quickstart-services-admin-ui>`
+
+Managing MetalK8s GUI, K8S and Salt API username and password
+-------------------------------------------------------------
+
+  .. warning::
+
+     #. In a typical **Metalk8s** cluster, ``/etc/kubernetes/htpasswd``
+        provides access control credentials to the following 3 components:
+        **MetalK8s GUI**, **K8S** and **Salt API**. Any changes made to this
```

Comment:
See comment above. This can be confusing, because

- what's a non-typical cluster and how would thing work there?
- That file only contains access credentials to the K8s API. UI doesn't have any creds/access control itself, and SaltAPI federates access to the K8s API.